modules/ocf_kubernetes/manifests/master/secrets.pp

```diff
@@ -0,0 +1,18 @@
+class ocf_kubernetes::master::secrets {
+
+  file {
+    default:
```

Comment:
we usually only use `default` when there is more then one file being created in the same resource definition